pom.xml

```diff
@@ -42,6 +42,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jenkins.version>1.651.2</jenkins.version>
```

Comment:
@rsandell `getAllParameters`, however, is new.

The cleaner solution would probably be to fail the `@Extension` (and make it optional) if the method doesn't exist, as it's redundant then.